modules/apps/headless/headless-delivery/headless-delivery-api/src/main/java/com/liferay/headless/delivery/dto/v1_0/CustomValue.java

```diff
@@ -110,7 +110,7 @@ public void setData_i18n(
 	@JsonProperty(access = JsonProperty.Access.READ_WRITE)
 	protected Map<String, String> data_i18n;
 
-	@Schema(description = "A point determined by latitude and longitude.")
+	@Schema
```

Comment:
We have lost the description of geo property? Did you try to add the reference with allOf to put it?